src/components/Tutorials/subComps/RichTextRenderer.jsx

```diff
@@ -0,0 +1,15 @@
+import { QuillDeltaToHtmlConverter } from 'quill-delta-to-html';
+import React from 'react'
+
+
+const RichTextRenderer = ({ delta }) => {
+    // Renders rich text from quill
+    var cfg = {};
+    var converter = new QuillDeltaToHtmlConverter(delta, cfg);
```

Comment:
Can we come up with a better variable name than `cfg` ? It is not making sense to me by reading the variable name 🤔 What if someone new contributor comes tomorrow to check codebase and he is not aware about `cfg`?  
What I understood by `cfg` is `config`? 

What are your thoughts?

Reply:
Actually I've taken this from official documentation of that package. So did not put thought into renaming it. I'll change the variable name.